src/schema.graphql

```diff
@@ -18,6 +18,7 @@ type Block {
   miner: Address!
   nonce: ByteString!
   previousBlock: Block
+  stateRootHash: ByteString!
```

Comment:
Isn't the `stateRootHash` of `type Block` nullable?

Reply:
Fixed it.